doc/tutorials/navigation2_on_real_turtlebot3.md

```diff
@@ -7,7 +7,7 @@
 
 # Overview:
 
-This tutorial shows how to control and navigate Turtlebot 3 using the ROS2 Navigation2 on a real Turtlebot 3 robot. Before completing this tutorials, completing the Navigation2 with Turtlebot 3 in Gazebo is highly recommended espesially if you are new to ROS and Navigation2.
+This tutorial shows how to control and navigate Turtlebot 3 using the ROS2 Navigation2 on a real Turtlebot 3 robot. Before completing this tutorials, completing the [Navigation2 with Turtlebot 3 in Gazebo](navigation2_with_turtlebot3_in_gazebo.md) is highly recommended espesially if you are new to ROS and Navigation2.
```

Comment:
You should add the link back to this tutorial in the other one as well.

Reply:
I put a step called the next tutorial at the end of the other tutorial. I linked it to this tutorial.